spec/models/enterprise_spec.rb

```diff
@@ -131,15 +131,15 @@
         e = Enterprise.new name: enterprise.name
         e.should_not be_valid
         e.errors[:name].first.should ==
-          "has already been taken. If this is your enterprise and you would like to claim ownership, please contact the current manager of this profile at owner@example.com."
+          "has already been taken. If this is your enterprise and you would like to claim ownership, or if you would like to trade with this enterprise please contact the current manager of this profile at %{email}."
```

Comment:
It's a bit more future proof if you use i18n here so that we use the translation key rather than the actual copy.